chrome/content/zotero/preferences/preferences_export.xul

```diff
@@ -66,7 +66,7 @@
 			
 			<separator/>
 			
-			<label value="&zotero.preferences.quickCopy.siteEditor.setings;" control="quickCopy-siteSettings"/>
+			<label value="&zotero.preferences.quickCopy.siteEditor.settings;" control="quickCopy-siteSettings"/>
```

Comment:
This would lose the strings on Transifex, so let's leave this and the corresponding `locale` entries as is for now.

Reply:
Hm... I fixed it in all the locals as well see https://github.com/zotero/zotero/pull/1570/commits/9649468042bdaa723710bee73911f42fbee2dce6 . Would we still lose something then?

Reply:
Well, I guess I can do a push to Transifex. I generally don't like to push non-source (i.e., non-English) strings, but if I do a pull immediately before it should be OK.